services/wallet/router/filter_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/ethereum/go-ethereum/common/hexutil"
 
 	"github.com/status-im/status-go/params"
-	"github.com/status-im/status-go/services/wallet/bridge"
+	pathprocessor "github.com/status-im/status-go/services/wallet/router/path_processor"
```

Comment:
shall u not just make the path pathprocessor?
align path and package?

Reply:
Yes, we can. Will do that now.